src/main/java/org/spongepowered/common/mixin/core/server/management/MixinPlayerInteractionManager.java

```diff
@@ -95,80 +98,6 @@ public void onBlockRemoving(final BlockPos pos, final CallbackInfo ci) {
         }
     }
 
-    /**
```

Comment:
And this has been tested for cancellations? This was added for bucket cancelations causing clients to think they had changed itemstacks.